packages/plugin-calendar/src/calendar.js

```diff
@@ -0,0 +1,32 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {SparkPlugin} from '@ciscospark/spark-core';
+import {pick} from 'lodash';
+
+const Calendar = SparkPlugin.extend({
+  namespace: `Calendar`,
+
+  /**
+   * Retrieves a collection of meetings based on the request parameters
+   * @param {Object} options
+   * @param {Date} options.fromDate the start of the time range
+   * @param {Date} options.toDate the end of the time range
+   * @returns {Promise} Resolves with an array of meetings
+   */
+  get(options) {
+    options = options || {};
+    return this.spark.request({
+      method: `GET`,
+      service: `calendar`,
+      resource: `calendarEvents`,
+      qs: pick(options, `fromDate`, `toDate`)
```

Comment:
is pick necessary? why not just forward all options as query params, that way when new ones get added, we don't need to release a new sdk.